api/server/server.go

```diff
@@ -1640,8 +1640,9 @@ func allocateDaemonPort(addr string) error {
 		return fmt.Errorf("failed to lookup %s address in host specification", host)
 	}
 
+	pa := portallocator.Get()
```

Comment:
@stevvooe. We do indeed use `sync.Once` for initializing the default instance. Just that we were only doing it once when `portallocator.New` was called. This way we don't need to do `once.Do` on all the methods. In fact this is how our original PR was, just naming this `New` but because of a previous comment we changed it to `Get`